doc/releases/release-notes-3.7.rst

```diff
@@ -188,6 +188,8 @@ Drivers and Sensors
 
 * Input
 
+* LoRa
+  * Added driver for Reyax LoRa module
```

Comment:
```suggestion
* LoRa

  * Added driver for Reyax LoRa module

```
You need an extra linebreak before and after so it renders properly 
https://builds.zephyrproject.io/zephyr/pr/69273/docs/releases/release-notes-3.7.html#drivers-and-sensors

Reply:
> You need an extra linebreak before and after so it renders properly https://builds.zephyrproject.io/zephyr/pr/69273/docs/releases/release-notes-3.7.html#drivers-and-sensors

done: https://github.com/zephyrproject-rtos/zephyr/blob/1a7cabc81eb467390d7262c29bfcce6b3394c2df/doc/releases/release-notes-3.7.rst